frontend/src/pushapi.spec.js

```diff
@@ -77,14 +77,14 @@ describe('PushApi', function () {
     beforeEach(function () {
       module(function (pushApiProvider) {
         pushApiProvider
-          .backendServiceUrl("/pushService/credentials");
+          .backendServiceUrl("/v1/pushService/credentials");
       });
 
       inject(function (_pushApi_, $httpBackend) {
         pushApi = _pushApi_;
         httpBackend = $httpBackend;
 
-        httpBackend.expectGET('/pushService/credentials').respond(200, {token: "token1", namespace: 'namespace'});
+        httpBackend.expectGET('/v1/pushService/credentials').respond(200, {token: "token1", namespace: 'namespace'});
```

Comment:
/v1/pushService/token sounds more reasonable as name as token is returned from that endpoint.

Reply:
You are right. Will rename.

Reply:
changed to /v1/pushService/token